containers/wordpress/wp_opt/https/files/nginx.conf.in

```diff
@@ -0,0 +1,125 @@
+worker_processes  96; # Depends on #vcpus
+
```

Comment:
missing "include  qat_module.so" in the wp_opt.

Reply:
done